examples/preview/sass/components/CTA.module.scss

```diff
@@ -0,0 +1,43 @@
+// CSS for the CTA component in components/CTA.tsx.
+
+@import "sass/variables";
```

Comment:
Nit: since you're using `.scss`, the more common folder naming that I've seen for these files seems to be `/scss` rather than '/sass`

Reply:
Thanks! Done in https://github.com/wpengine/headless-framework/pull/109/commits/b46b61eb69c04b930402d8a1e9a68d2df23ce662.